backend/open_webui/utils/chat.py

```
@@ -40,7 +40,10 @@
 from open_webui.models.models import Models
 
 
-from open_webui.utils.plugin import load_function_module_by_id
+from open_webui.utils.plugin import (
+    load_function_module_by_id,
```

Ruff (0.11.9)

`open_webui.utils.plugin.load_function_module_by_id` imported but unused

Remove unused import: `open_webui.utils.plugin.load_function_module_by_id`

(F401)